build/ci/e2e/Jenkinsfile-new

```diff
@@ -29,7 +29,7 @@ pipeline {
                     cat >operators/run-config.yml <<EOF
 id: gke-ci
 overrides:
-  kubernetesVersion: $CLUSTER_VERSION
+  kubernetesVersion: "$CLUSTER_VERSION"
```

Comment:
I had trouble understanding what's going on until I look at the log of the last job: "_error type mismatch on KubernetesVersion field: found float64, expected string_". 
=> Ok!